diff --git a/VuhDo.lua b/VuhDo.lua
index af77066..48f207b 100644
--- a/VuhDo.lua
+++ b/VuhDo.lua
@@ -385,8 +385,11 @@ function VUHDO_setHealth(aUnit, aMode)
 
 			if tIsDcChange then VUHDO_updateBouquetsForEvent(aUnit, 19); end-- VUHDO_UPDATE_DC
 
-			if 2 == aMode then -- VUHDO_UPDATE_HEALTH
+			if 2 == aMode or 12 == aMode then -- VUHDO_UPDATE_HEALTH -- VUHDO_UPDATE_HEALTH_COMBAT_LOG
 				tNewHealth = UnitHealth(aUnit);
+				if 12 == aMode then -- VUHDO_UPDATE_HEALTH_COMBAT_LOG
+					tNewHealth = tInfo["loghealth"];
+				end
 				if not tIsDead and tInfo["health"] > 0 then
 					tInfo["lifeLossPerc"] = tNewHealth / tInfo["health"];
 				end
@@ -406,6 +409,7 @@ function VUHDO_setHealth(aUnit, aMode)
 				tInfo["dead"] = tIsDead;
 				tInfo["healthmax"] = UnitHealthMax(aUnit);
 				tInfo["sortMaxHp"] = VUHDO_getUnitSortMaxHp(aUnit);
+				tInfo["loghealth"] = UnitHealth(aUnit);
 
 			elseif 6 == aMode then -- VUHDO_UPDATE_AFK
 				tInfo["afk"] = tIsAfk;
@@ -435,7 +439,7 @@ function VUHDO_updateHealth(aUnit, aMode)
 
 	tIsPet = VUHDO_RAID[aUnit] and VUHDO_RAID[aUnit]["isPet"];
 
-	if not tIsPet or VUHDO_INTERNAL_TOGGLES[26] then -- VUHDO_UPDATE_PETS  -- Enth„lt nur Pets als eigene Balken, vehicles werden ?ber owner dargestellt s.unten
+	if not tIsPet or VUHDO_INTERNAL_TOGGLES[26] then -- VUHDO_UPDATE_PETS  -- Enth\84lt nur Pets als eigene Balken, vehicles werden ?ber owner dargestellt s.unten
 		VUHDO_setHealth(aUnit, aMode);
 		VUHDO_updateHealthBarsFor(aUnit, aMode);
 	end
@@ -450,7 +454,7 @@ function VUHDO_updateHealth(aUnit, aMode)
 	end
 
 	if 1 ~= sCurrentMode -- VUHDO_MODE_NEUTRAL
-		and (2 == aMode or 3 == aMode) then -- VUHDO_UPDATE_HEALTH -- VUHDO_UPDATE_HEALTH_MAX
+		and (2 == aMode or 3 == aMode or 12 == aMode) then -- VUHDO_UPDATE_HEALTH -- VUHDO_UPDATE_HEALTH_MAX --  VUHDO_UPDATE_HEALTH_COMBAT_LOG
 		-- Remove old emergencies
 		VUHDO_FORCE_RESET = true;
 		for tUnit, _ in pairs(VUHDO_EMERGENCIES) do
diff --git a/VuhDoCombatLogParser.lua b/VuhDoCombatLogParser.lua
index 37f3a9b..5695e81 100644
--- a/VuhDoCombatLogParser.lua
+++ b/VuhDoCombatLogParser.lua
@@ -35,8 +35,8 @@ local function VUHDO_addUnitHealth(aUnit, aDelta)
 		elseif tNewHealth > tInfo["healthmax"]  then tNewHealth = tInfo["healthmax"]; end
 
 		if tInfo["health"] ~= tNewHealth then
-			tInfo["health"] = tNewHealth;
-			VUHDO_updateHealth(aUnit, 2); -- VUHDO_UPDATE_HEALTH
+			tInfo["loghealth"] = tNewHealth;
+			VUHDO_updateHealth(aUnit, 12); -- VUHDO_UPDATE_HEALTH_COMBAT_LOG
 		end
 	end
 end
diff --git a/VuhDoConst.lua b/VuhDoConst.lua
index 31344fe..489e642 100644
--- a/VuhDoConst.lua
+++ b/VuhDoConst.lua
@@ -252,6 +252,7 @@ VUHDO_UPDATE_TARGET = 8;
 VUHDO_UPDATE_INC = 9;
 VUHDO_UPDATE_ALIVE = 10;
 VUHDO_UPDATE_EMERGENCY = 11;
+VUHDO_UPDATE_HEALTH_COMBAT_LOG = 12;
 -- nur für bouquets
 VUHDO_UPDATE_MANA = 13;
 VUHDO_UPDATE_THREAT_PERC = 14;
